docs/python/source/conf.py

```diff
@@ -0,0 +1,53 @@
+"""Configuration file for the Sphinx documentation builder."""
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../../"))
+
+# from rustyms import __version__
+
+# Project information
+project = "rustyms"
+author = "Douwe Schulte, Ralf Gabriels"
+github_project_url = "https://github.com/compomics/rustyms/"
```

Comment:
At some point these can be updated ;-)

Reply:
Whoops. `compomics` was a copy-paste error from another project.